torchaudio/prototype/models/conformer_wav2vec2.py

```diff
@@ -0,0 +1,290 @@
+from typing import List, Optional, Tuple, Union
```

Comment:
Could you prefix the file with underscore so as to emphasis that client code should import model from `torchaudio.[prototype.]models`?